crates/ruff_python_formatter/src/expression/expr_name.rs

```diff
@@ -11,16 +11,11 @@ pub struct FormatExprName;
 
 impl FormatNodeRule<ExprName> for FormatExprName {
     fn fmt_fields(&self, item: &ExprName, f: &mut PyFormatter) -> FormatResult<()> {
-        let ExprName { id, range, ctx: _ } = item;
-
-        debug_assert_eq!(
-            id.as_str(),
-            f.context()
-                .source_code()
-                .slice(*range)
-                .text(f.context().source_code())
-        );
```

Comment:
This debug assertion no longer passes as-is, since the AST node holds the NFKC-normalized identifier, whereas the source code gives us the original identifier.

If we think this assertion is valuable, we could put it in a `#[cfg(test)]` block, make the `unicode-normalization` crate a test-dependency of the formatter, and assert that the NFKC-normalized source code is equal to the AST `id`. I don't know if that's really worth it, though?

Reply:
I think it's fine to remove it.